src/app/activity/bounty/Claim.ts

```diff
@@ -68,56 +100,147 @@ const getDbHandler = async (request: ClaimRequest): Promise<{dbBountyResult: Bou
     }
 }
 
-const writeDbHandler = async (request: ClaimRequest, claimedByUser: GuildMember): Promise<void> => {
+const writeDbHandler = async (request: ClaimRequest, dbBountyResult: BountyCollection, claimedByUser: GuildMember): Promise<BountyCollection> => {
```

Comment:
can we consider renaming `dbBountyResult` to something including the word parent?

Reply:
It is only a parent in the evergreen case. writeDbHandler is used for both evergreen and not.